doc/tutorials/rt_linux.rst

```diff
@@ -44,71 +44,67 @@ system on Intel KBL NUC with a SATA SSD as ``/dev/sda`` and an NVME SSD as
-         $ git apply rt_linux.patch 
-         $ xzcat ../patch-4.19.31-rt18.patch.xz | patch -p1
-
-   #. Edit the ``kernel_config_uos`` config file: search for the keyword
-      "NVME Support", delete ``# CONFIG_BLK_DEV_NVME is not set`` and add two lines under "NVME Support" to enable
-      the NVME driver in RT kernel::
-
-         CONFIG_NVME_CORE=y
-         CONFIG_BLK_DEV_NVME=y
-
-   #. Build the RT kernel::
-
-         $ cp kernel_config_uos .config
-         $ make targz-pkg
-
-      Choose "Fully Preemptible Kernel (RT)" when prompted, and
-      choose default for all the other options.
-
-   #. Copy the generated package to SOS::
-
-         $ scp linux-4.19.28-rt18-quilt-2e5dc0ac-dirty-x86.tar.gz <user name>@<SOS ip>:~/
-
-#. Configure the system on SOS
-
-
-   a. Extract kernel boot and lib modules from the package::
-
-         $ cd ~/
-         $ tar xzvf linux-4.19.28-rt18-quilt-2e5dc0ac-dirty-x86.tar.gz
-
-   #. Copy the extracted lib modules to NVME SSD::
-
-         $ mount /dev/nvme0n1p3 /mnt
-         $ cp -r ~/lib/modules/4.19.28-rt18-quilt-2e5dc0ac-dirty /mnt/lib/modules
-
-   #. Edit and run the ``launch_hard_rt_vm.sh`` script to launch the UOS.
-      A sample ``launch_hard_rt_vm.sh`` is included in the Clear Linux
-      release, and is also available in the acrn-hypervisor/devicemodel
-      GitHub repo (in the samples folder).
-
-      You'll need to modify two places:
-
-      1. Replace ``/root/rt_uos_kernel`` with ``~/boot/vmlinuz-4.19.28-rt18-quilt-2e5dc0ac-dirty``
-      #. Replace ``root=/dev/sda3`` with ``root=/dev/nvme0n1p3``
-
-   #. Run the launch script::
-
-         $ sudo ./launch_hard_rt_vm.sh
+#. Set up and launch RT-LaaG
+
+   a. Add kernel-lts2018-preempt-rt bundle from CL (root)::
+
+         # swupd bundle-add kernel-lts2018-preempt-rt
+
+   #. Copy preempt-rt module to NVMe disk::
+
+         # mount /dev/nvme0n1p3 /mnt
+         # ls /usr/lib/modules/
+         # 4.19.31-5.iot-lts2018-preempt-rt/ 4.19.36-48.iot-lts2018/           4.19.36-48.iot-lts2018-sos/       5.0.10-7.preempt_rt/              5.0.14-753.native/
+         # cp -r /usr/lib/modules/4.19.31-6.iot-lts2018-preempt-rt /mnt/lib/modules/
+         # cd ~ && umount /mnt && sync
+
+   #. Modify launch_hard_rt_vm.sh script::
+
+         # vim /usr/share/acrn/samples/nuc/launch_hard_rt_vm.sh
+         <Get NVMe pass-through IDs>
+         # lspci -nn | grep SSD
+         01:00.0 Non-Volatile memory controller [0108]: Intel Corporation SSD Pro 7600p/760p/E 6100p Series [8086:f1a6] (rev 03)
+      
+         <Modify below to NVMe pass-through>
+         echo "8086 9d03" > /sys/bus/pci/drivers/pci-stub/new_id
+         echo "0000:00:17.0" > /sys/bus/pci/devices/0000:00:17.0/driver/unbind
+         echo "0000:00:17.0" > /sys/bus/pci/drivers/pci-stub/bind
+      
+         TO:
+         echo "8086 f1a6" > /sys/bus/pci/drivers/pci-stub/new_id
```

Comment:
Are these new values the same for everyone, or are these examples and you should use the output from the ``lspci`` command above?  (You don't say these values could be different on your system.)